src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -441,6 +447,13 @@ public boolean isOpen() {
     @Override
     public void close() throws IOException {
         if (open) {
+            exec.shutdown();
+            try {
+                exec.awaitTermination(3, TimeUnit.MINUTES);
```

Comment:
David suggests `awaitTermination` first, and then if it timed out, calling `shutdownNow`.  Lets go with maybe 10 seconds?  Seems like a reasonable amount of time that no one will be sad waiting for it but most transfers should finish cleanly.